File: 0_Trabalho_Pratico_Padaria/Bakehouse/Bakehouse.Domain/Interfaces/IOrderPadItem.cs
using Bakehouse.Domain.Entities;
using FluentResults;
using System.Collections.Generic;

namespace Bakehouse.Domain.Interfaces
{
    public interface IOrderPadItem
    {
        public Result Insert(OrderPadItem item);
        public OrderPadItem FindById(int id);
        public IEnumerable<OrderPadItem> FindAll();
        public Result Delete(int id);
        public Result Update(int id, OrderPadItem item);
    }
}

File: 0_Trabalho_Pratico_Padaria/BakehouseJpaHB/src/main/java/com/bakehouse/dao_interfaces/IOrderPadItem.java
package com.bakehouse.dao_interfaces;

import com.bakehouse.domain.OrderPad;
import com.bakehouse.domain.OrderPadItem;
import java.util.List;

public interface IOrderPadItem {
    
    public List<OrderPadItem> findByIdOrderPad(OrderPad orderPad);
}